<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupDialog</class>
 <widget class="QDialog" name="GroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a group â€“ Coquery</string>
  </property>
  <layout class="QGridLayout" name="verticalLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Group name:</string>
     </property>
     <property name="buddy">
      <cstring>edit_label</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edit_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Columns:</string>
     </property>
     <property name="buddy">
      <cstring>widget_selection</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CoqListSelect" name="widget_selection"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_duplicates">
     <property name="text">
      <string>Duplicates:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radio_keep_duplicates">
       <property name="text">
        <string>&amp;Keep</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_remove_duplicates">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Functions:</string>
     </property>
     <property name="buddy">
      <cstring>scroll_content</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QScrollArea" name="scroll_area">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scroll_content">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>514</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scroll_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoqListSelect</class>
   <extends>QListWidget</extends>
   <header location="global">..listselect</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_label</tabstop>
  <tabstop>widget_selection</tabstop>
  <tabstop>radio_keep_duplicates</tabstop>
  <tabstop>scroll_area</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
